src/app/modules/group/components/managed-group-list/managed-group-list.component.ts

```diff
@@ -17,6 +17,10 @@ export class ManagedGroupListComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    this.fetchData();
+  }
+
+  fetchData(): void {
     this.managedGroupService.getManagedGroups().subscribe({
```

Comment:
I think there should be a leading `this.state = 'loading'` before the actual fetch because when there's an error and we click on the refresh button, there is no loading state.
It's particularly visible when you have a bad network connection.
(For instance, using Chromium dev tools with network throttling preset "Slow 3G" does the trick)